.pre-commit-config.yaml

```diff
@@ -0,0 +1,67 @@
+repos:
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        name: Formatting - isort
+        args:
+          [
+            "--profile",
+            "black",
+            "-l",
+            "99",
+            "multi_line_output",
+            "3",
+            "src/tmi",
+          ]
+  - repo: local
+    hooks:
+      - id: black
```

Comment:
Is `black` the best formatter to use for our repos? It's great but also on the stricter side. Any particular advantages it offers over other formatters (ex. `autopep8`) for our project?

Reply:
Personally, I like black because it's plug and play and the cost of being uncompromising is something that I'm ok with, but autopep8 works well too. There aren't really any advantages between any of them, all of these are options that work well, none are solutions however.